docker-compose.yml

```diff
@@ -1,53 +1,83 @@
 version: '2'
 services:
 
-  postgres:
-    image: postgres
+  database:
+    image: camptocamp/postgis:9.5
```

Comment:
GeoNode has two different databases one for configuration data and another for spatial data from postgis tables as datasource for geoserver